index.html

```diff
@@ -114,27 +113,76 @@ <h3>LET'S CONNECT</h3>
       </section>
       <section>
         <div class="nnjjn">
-          <h3 class="text-languages">Languages</h3><img src="IMG/arrDown.png" alt="arrdown" class="arrdown">
-        </div><br>
+          <h3 class="text-languages">Languages</h3>
+          <img src="IMG/arrDown.png" alt="arrdown" class="arrdown" />
+        </div>
+        <br />
         <div class="round-img-list">
           <div class="image-coding">
-            <img src="IMG/js.svg" alt="" class="round-img" /> <p class="text-language-image">JavaSript</p><br>
+            <img src="IMG/js.svg" alt="" class="round-img" />
+            <p class="text-language-image">JavaSript</p>
+            <br />
           </div>
           <div class="image-coding">
-            <img src="IMG/html.svg" alt="" class="round-img" /> <p class="text-language-image">HTML</p><br>
+            <img src="IMG/html.svg" alt="" class="round-img" />
+            <p class="text-language-image">HTML</p>
+            <br />
           </div>
           <div class="image-coding">
-            <img src="IMG/css.svg" alt="" class="round-img" /> <p class="text-language-image">CSS</p>
+            <img src="IMG/css.svg" alt="" class="round-img" />
+            <p class="text-language-image">CSS</p>
           </div>
-        </div><br>
+        </div>
+        <br />
         <div class="nnjjn a">
-          <h3 class="text-languages">Frameworks</h3><img src="IMG/Union.svg" alt="arrright" class="arrright">
+          <h3 class="text-languages">Frameworks</h3>
+          <img src="IMG/Union.svg" alt="arrright" class="arrright" />
         </div>
         <br />
         <div class="nnjjn a">
-          <h3 class="text-languages">Skills</h3><img src="IMG/Union.svg" alt="arrright" class="arrright">
+          <h3 class="text-languages">Skills</h3>
+          <img src="IMG/Union.svg" alt="arrright" class="arrright" />
         </div>
       </section>
     </div>
+    <div class="container_contact">
+      <div class="content">
+        <h1 class="h1_contact">Contact me!</h1>
```

Comment:
`h1` tags are used for main tiltes (first heading of a web page) and this is not a main title so `h2`,`h3`, etc can be used visit this [Link](https://www.w3schools.com/tags/tag_hn.asp) for more information on HTML headings